release-notes.md

```diff
@@ -1,19 +1,49 @@
+2.0.0-RC7
+---------
+* CI/CD implementation
+* Delivery Tracker - limit requests map per peer
+* add request tracker actor
+* add upper bound limit for peers database
+* rename Scorex -> Sparkz
+* update lib versions to latest, check and fix vulnerabilities
+* KnownPeers should never be discarded
+* exclude unconfirmedConnections and already active connections
```

Comment:
Perhaps we can add some context here: e.g. `exclude unconfirmedConnections and already active connections when selecting a new candidate peer to connect to`